test/evergreen/print_stack_trace.py

```diff
@@ -130,24 +130,24 @@ def main():
 
         # Get the executable from the core file itself.
         proc = subprocess.Popen(["file", core_file_path], stdout=subprocess.PIPE)
-        executable_path = str(proc.communicate())
-        start = executable_path.find('execfn: ')
-        end = executable_path.find('platform:')
-        executable_path = executable_path[(start+9):(end-3)]
+        # The file command prints something similar to:
+        # WT_TEST/test_practice.0/dump_python3.16562.core: ELF 64-bit LSB core file x86-64, version 1 (SYSV),
+        # SVR4-style, from 'python3 /home/ubuntu/wiredtiger/test/suite/run.py -j 1 -p test_practice',
+        # real uid: 1000, effective uid: 1000, real gid: 1000, effective gid: 1000, execfn: '/usr/bin/python3', platform: 'x86_64'
+        output = str(proc.communicate())
 
-        # Extract the filename from the core file path, to create a stacktrace output file.
-        file_name, _ = os.path.splitext(os.path.basename(core_file_path))
-        file_name = file_name + ".stacktrace.txt"
+        # The field of interest is execfn: '/usr/bin/python3' from the example above.
+        start = output.find('execfn: ')
+        # Fetch the value of execfn. This is the executable path!
+        executable_path = re.search(r'\'.*?\'', output[start:]).group(0)
```

Comment:
We need to make sure the `search` call returns something valid, otherwise, we should gracefully fail? But if we do it this way, we might not ever find out that we are no longer finding the executable.

Reply:
Etienne and I discussed and we plan to keep it the way it is; as it may rarely fail due to this reason and if it does, it is good to not fail gracefully.